app/views/components/_published-dates.html.erb

```diff
@@ -7,21 +7,20 @@
   history_class = "app-c-published-dates--history" if history.any?
 %>
 <% if published || last_updated %>
-<div class="app-c-published-dates <%= history_class %>" <% if history.any? %>id="history" data-module="toggle"<% end %>>
+<div class="app-c-published-dates <%= history_class %>" <% if history.any? %>id="history" data-module="toggle"<% end %> lang="en">
   <% if published %>
-    <%= t('components.published_dates.published', date: published) %>
+    Published <%= published %>
   <% end %>
   <% if last_updated %>
-    <% if published %><br /><% end %><%= t('components.published_dates.last_updated', date: last_updated) %>
+    <% if published %><br /><% end %>Last updated <%= last_updated %>
     <% if link_to_history && history.empty? %>
-      &mdash; <a href="#history"><%= t('components.published_dates.see_all_updates') %></a>
+      &mdash; <a href="#history">see all updates</a>
     <% elsif history.any? %>
       <a href="#full-history"
       class="app-c-published-dates__toggle"
       data-controls="full-history"
       data-expanded="false"
-      data-toggled-text="&#45;&nbsp;<%= t('components.published_dates.hide_all_updates') %>">&#43;&nbsp;<%= t('components.published_dates.show_all_updates')
-       %></a>
+      data-toggled-text="&#45;&nbsp;hide all updates">&#43;&nbsp;show all updates</a>
```

Comment:
As above, we can specify the locale in the translation by passing the `locale: :en` param.